definitions/procedures/content/prepare.rb

```diff
@@ -8,7 +8,10 @@ class Prepare < ForemanMaintain::Procedure
     def run
       sleep(20) # in satellite 6.9 the services are still coming up
       # use interactive to get realtime output
-      puts execute!('foreman-rake katello:pulp3_migration', :interactive => true)
+      puts execute!(
+        'foreman-rake katello:pulp3_migration preserve_output=true',
```

Comment:
You won't need this with my PR here: https://github.com/theforeman/foreman_maintain/pull/516